packages/manager/modules/pci/src/projects/project/additional-ips/additional-ips.html

```diff
@@ -0,0 +1,135 @@
+<div data-ui-view>
+    <oui-header
+        data-heading="{{:: 'pci_additional_ips_title' | translate }}"
+        class="mb-3"
+    >
+        <oui-guide-menu
+            data-text="{{:: 'pci_project_guides_header' | translate }}"
+        >
+            <oui-guide-menu-item href="{{ $ctrl.guideUrl }}" data-external>
+                <span
+                    data-translate="pci_project_guides_header_all_guides"
+                ></span>
+            </oui-guide-menu-item>
+        </oui-guide-menu>
+    </oui-header>
+    <div>
+        <p data-translate="pci_additional_ips_additional_ips_description"></p>
+        <pci-maintenance-banner
+            data-steins="$ctrl.steins"
+            data-customer-regions="$ctrl.customerRegions"
+            data-product-regions="$ctrl.additionalIpsRegions"
+            data-is-list-page="true"
+            data-product-name=":: 'pci_additional_ips_title' | translate"
+        ></pci-maintenance-banner>
+
+        <pci-announcement-banner
+            data-get-state-name="$ctrl.getStateName()"
+            data-on-go-to-region="$ctrl.goToRegion()"
+        ></pci-announcement-banner>
+
+        <cui-message-container
+            data-messages="$ctrl.messages"
+        ></cui-message-container>
+
+        <oui-tabs>
+            <oui-tabs-item
+                heading="{{:: 'pci_additional_ips_failover_ip_title' | translate}}"
+            >
+                <oui-datagrid
+                    data-rows="$ctrl.additionalIps"
+                    criteria="$ctrl.criteria"
+                    on-criteria-change="$ctrl.onListParamChange($criteria)"
+                >
+                    <oui-datagrid-topbar>
+                        <oui-action-menu
+                            data-text="{{:: 'pci_additional_ips_actions' | translate }}"
+                        >
+                            <oui-action-menu-item
+                                state="pci.projects.project.additional-ips.imports"
+                            >
+                                <span
+                                    data-translate="pci_additional_ips_import_failover_ip"
+                                ></span>
+                            </oui-action-menu-item>
+                            <oui-action-menu-item
+                                data-on-click="$ctrl.goToAdditionalIpOrderPage()"
+                            >
+                                <span
+                                    data-translate="pci_additional_ips_order"
+                                ></span>
+                            </oui-action-menu-item>
+                        </oui-action-menu>
+                    </oui-datagrid-topbar>
+                    <oui-datagrid-column
+                        data-title=":: 'pci_additional_ips_failover_ip_title' | translate"
+                        data-property="ip"
```

Comment:
we display to times `id` property ?

Reply:
No. May be you got confused between IP and ID?

Reply:
true :)